saphanabootstrap-formula.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Friday Dec 04 11:27:52 UTC 2022 - Steve Stringer <steven.stringer@suse.com>
```

Comment:
Seems you computer is currently running in a parallel time line :-) My calendar tells me, we are still in November and not December.
Even that it is not a major problem it will be nice, if you can fix it.

Reply:
Good spot, thanks.

Corrected and pushed.